src/cascadia/TerminalApp/MinMaxCloseControl.cpp

```diff
@@ -78,6 +78,16 @@ namespace winrt::TerminalApp::implementation
             MaximizeButton().Height(maximizedHeight);
             CloseButton().Height(maximizedHeight);
             break;
+        case WindowVisualState::WindowVisualStateUnfocused:
+            VisualStateManager::GoToState(MaximizeButton(), L"WindowStateUnfocused", false);
+            VisualStateManager::GoToState(MinimizeButton(), L"WindowStateUnfocused", false);
+            VisualStateManager::GoToState(CloseButton(), L"WindowStateUnfocused", false);
+            break;
+        case   WindowVisualState::WindowVisualStateFocused:
+            VisualStateManager::GoToState(MaximizeButton(), L"WindowStateFocused", false);
+            VisualStateManager::GoToState(MinimizeButton(), L"WindowStateFocused", false);
+            VisualStateManager::GoToState(CloseButton(), L"WindowStateFocused", false);

```

Comment:
I'm worried about this -- focus _stacks with_ maximized/normal, it doesn't replace it. Be careful!

Reply:
Yeah, I was wondering the same. So, I should let the focused case fall through and move the VSM statements to the end, correct?

Reply:
yeah.. might be best to disentangle focused from visual state ... _or_ make visual state a bitfield(!)

Reply:
> make visual state a bitfield(!)

If only I knew how to :)

Reply:
Ugh! Great, now there are 2 reviews with the exact same text! Github can be weird